spec/support/seeds/seeds_e2e.rb

```diff
@@ -42,6 +41,12 @@
     checked_terms_and_conditions: true,
   )
 
+  user.profile.update({
```

Comment:
Nitpick (blocking): Instead of running an update, I imagine we can add these properties for the user just above where the summary was removed?

Reply:
I initially coded it like that, but the data didn't reflect in the UI (including the summary that was there before). Not sure if there's some other way to do it to save the update call 🤔

Reply:
I double-checked and yeah unfortunately we can't create both a User and a Profile with attributes all in the same step, so we need to have two actions (I guess it's only in test setup we'd want to do both at the same time)